_data/news.yml

```diff
@@ -1,6 +1,10 @@
+- name: pysal 2.2.0
+  date: 2020-02-13
+  summary: <a href="https://pysal.org/pysal/">pysal 2.2.0</a> released on <a href="https://pypi.org/project/pysal/2.2.0/">PyPI</a>. <a href="https://github.com/pysal/pysal/releases/tag/v2.2.0">Release Notes</a>.
```

Comment:
should we add "are here" like in the `esda` below?

Reply:
I cut it out as it is redundant.

Reply:
OK